CHANGELOG.md

```diff
@@ -2,12 +2,97 @@
 
 ## [Unreleased](https://github.com/rerun-io/rerun/compare/latest...HEAD)
 
+## [0.11.0](https://github.com/rerun-io/rerun/compare/0.10.1...0.11.0)
+
+### Overview & Highlights
+
+- 🌊 C++ SDK improvements
+  - [reference docs are live!](https://ref.rerun.io/docs/cpp/)
+  - 2x-5x faster logging
+  - CMake install support and other CMake setup improvements
+  - Support for custom components & archetypes
+  - Zero copy logging for images, various API improvements
+- 📈 Visual History improvements
+  - Time series plots can now limit its query to a range
+  - Much more powerful UI, allowing query ranges relative to time cursor
+- 🕸️ The viewer can now be easily embedded in your web apps via our [npm package](https://www.npmjs.com/package/@rerun-io/web-viewer)
+- 🐍 ⚠️ legacy Python API now removed, check the [migration guide](https://github.com/rerun-io/rerun/issues/723) if you're not using `rr.log` yet
+- 🦀 `StoreView` allows to track changes to the store. This is groundwork for upcoming performance improvements (but if you customize the viewer you have access to it as well!)
+
+⚠️ Known issues on Visual History:
```

Comment:
ah we renamed it already now? good

Reply:
Yeah it's been changed since the big PR

<img width="390" alt="image" src="https://github.com/rerun-io/rerun/assets/49431240/7215c42f-efb7-48e9-90a2-a9f9182b3ac7">

Reply:
Oh right, it should be "Visible Time Range" then